pycobertura/reporters.py

```diff
@@ -234,6 +244,38 @@ def generate(self):
         return tabulate(lines, headers=headers)
 
 
+class CsvReporterDelta(DeltaReporter):
+    not_available = "-"
+
+    def generate(self, delimiter):
```

Comment:
I understand that - maybe I can refactor that to make it more understandable. Unfortunately I did not find an easy way yet, because in the show_source case we have to append the "Missing" item to every list inside a list.

Reply:
We don't need to refactor, but if you leave comments describing the input/output such as:

```python
# Example:
# lines.values() == [[1, 2, 3], [4, 5, 6]]
# lines == [...]
lines = [[f"{item}" for item in row[:-1]] for row in zip(*lines.values())]
```

Something like that. Does that make sense?

Reply:
Can you re-view again with the recent commit: https://github.com/aconrad/pycobertura/pull/130/commits/04ea485ffa6c116aebf0f20aa3b88f9bb037be3e.

It should already be much clearer :)

Reply:
Updated in https://github.com/aconrad/pycobertura/pull/130/commits/7e158c128b32ecb88f63baef9559e5b5eab1bbea